README.md

```diff
@@ -15,20 +15,22 @@ Font Bakery is not an official Google project, and Google provides no support fo
 
 ## Web Usage
 
-Visit [fontbakery.appspot.com](https://fontbakery.appspot.com) and drop 1 to 18 TTF files on the page, and click the "3"
+To use Font Bakery through a web UI is currently not supported. 
 
-The source code to this web-based user-interface is all in the `webapp/` folder.
-In order to run it on your own computer, download this source repo and all dependencies for command line usage.
-Then run the following commands:
+A web dashboard is under development at <https://github.com/googlefonts/fontbakery-dashboard/> and will allow monitoring the check results of collections of font families, such as the entire Google Fonts collection.
 
-    cd webapp
-    ./dev_appserver.py app
-
-Now open <http://0.0.0.0:8000> in your browser.
+This tool was initialy developed in this repository, but later it was split out into its own git repo.
 
 ## Command Line Usage
 
-Fontbakery installs its own command `fontbakery` in your `$PATH`. The `fontbakery` command makes other subcommands accessible. Here's the output of the command line help:
+For infrequent usage, clone this repo and then run each tool by itself:
+
+    PYTHONPATH=Lib python bin/fontbakery-metadata-vs-api.py <args>
+
+For regular usage, install Font Bakery as a package, following the [installation instructions](https://github.com/googlefonts/fontbakery#install) below.
+This puts thex `fontbakery` command in your `$PATH`, and makes all other subcommands accessible. 
```

Comment:
`s/thex/the/`

Also, it put's all the `fontbakery-{subcommands}` in your path. We should think about a nicer solution.